backend/reinvestmentpro/settings.py

```diff
@@ -59,13 +59,20 @@
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+CORS_ALLOWED_ORIGINS = [
+    "https://example.com", #domain
+    "https://sub.example.com", #subdomain
+    "http://localhost:5173", #development server
```

Comment:
In your `vite.config` on your react project you can specify which server and what port you want your project running in for consistency purpose. You just need to add the following to vite.confic:
```js
// vite.config.js (or vite.config.ts)
export default defineConfig({
  server:{
    host:"0.0.0.0",
    port:5173,
  },
  plugins: [react()],
})
```
Definitely not a must do but it may help if you're specifying ALLOWED_ORIGINS with CORS